sqlglot/dialects/clickzetta.py

```diff
@@ -242,6 +297,8 @@ class Generator(Spark.Generator):
             exp.DateAdd: date_add_sql,
             exp.RegexpExtract: regexp_extract_sql,
             exp.DayOfWeek: adjust_day_of_week,
+            exp.Group: _group_sql,
+            exp.Kill: lambda self, e: f"CANCEL JOB {self.sql(e.this)}",
```

Comment:
Not very necessary to transpile a cancel job here. Our job id is auto generated, even a exp.Kill is transpiled, it has no change to run correctly.